database/migrations/2019_07_02_191541_add_translated_name_and_impact_to_departments.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+use App\Models\Lookup\Department;
+
+class AddTranslatedNameAndImpactToDepartments extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('departments', function (Blueprint $table) {
+            $table->json('name')->nullable();
+            $table->json('impact')->nullable();
+        });
+
+        $departments = Department::all();
```

Comment:
Its a bad idea to reference the Department model in a migration, because the model could change later, breaking this migration. Its better to use raw db queries. For an example, take a look at database\migrations\2019_06_11_220005_convert_existing_work_env_and_culture_to_job_poster.php

Reply:
Refactored.